### ISSUE TITLE
RollingFileAppender and FixedWindowRollingPolicy are broken.

### ISSUE THREAD

**Issue by crocket** [pyr/unilog#1]
The config is

```
{:level :debug
  :console true
  :appenders [{:appender :rolling-file
                           :rolling-policy {:type :fixed-window, :max-index 5}
                           :triggering-policy {:type :size-based, :max-size 51200}
                           :file "clj-ddns-client.log"}]}
```

Below is the stacktrace.

```
Unhandled java.lang.NullPointerException
   (No message)

        RollingPolicyBase.java:   94  ch.qos.logback.core.rolling.RollingPolicyBase/getParentsRawFileProperty
 FixedWindowRollingPolicy.java:  167  ch.qos.logback.core.rolling.FixedWindowRollingPolicy/getActiveFileName
      RollingFileAppender.java:  122  ch.qos.logback.core.rolling.RollingFileAppender/getFile
      RollingFileAppender.java:   84  ch.qos.logback.core.rolling.RollingFileAppender/start
 NativeMethodAccessorImpl.java:   -2  sun.reflect.NativeMethodAccessorImpl/invoke0
 NativeMethodAccessorImpl.java:   62  sun.reflect.NativeMethodAccessorImpl/invoke
DelegatingMethodAccessorImpl.java:   43  sun.reflect.DelegatingMethodAccessorImpl/invoke
                   Method.java:  497  java.lang.reflect.Method/invoke
                Reflector.java:   93  clojure.lang.Reflector/invokeMatchingMethod
                Reflector.java:  313  clojure.lang.Reflector/invokeNoArgInstanceMember
                    config.clj:  314  unilog.config/start-logging!
                      core.clj:  117  clj-ddns-client.core/-main/fn
                      core.clj:   64  clj-ddns-client.core/handle-cli-help!
                      core.clj:  112  clj-ddns-client.core/-main
                   RestFn.java:  408  clojure.lang.RestFn/invoke
                          REPL:    1  clj-ddns-client.core/eval20887
```

**Comment by crocket** [comment]
It is caused because FixedWindowRollingPolicy's parent is set to null.
The parent should be set to the parent RollingFileAppender.

**Comment by pyr** [comment]
Fixed in #2
